adapter/src/main/resources/Dockerfile

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 # -----------------------------------------------------------------------
 
-FROM alpine:3.19.1
+FROM alpine:3.21.0
```

Comment:
Can't we update enforcer base image as well ?

Reply:
Ack. Shall I update this with the rate limiter Docker image update PR?